src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -91,31 +91,31 @@
            (if editing?
              (list
               [buttons/SidebarButton
-               {:color :success-state
+               {:data-test-id "save-editted-method-config-button"
```

Comment:
nit because this would annoy me forever: "edited" (spelling). But ... this might require test changes if anything uses this.